.github/workflows/sfl-pr-review.md

```
@@ -0,0 +1,184 @@
+    allowed-events: [APPROVE, REQUEST_CHANGES]
+    supersede-older-reviews: true
+    footer: always
+  create-check-run:
+    max: 1
+    name: "SFL Reviewer Approval"
+  remove-labels:
+    allowed: [sfl-review]
+    max: 1
+    target: triggering
+---
+# Deployed from: HemSoft/set-it-free-loop/deployment/workflows/sfl-pr-review.md@235a0df0c70fded4982e8763223e7c9936242215
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)